gprofiler/metadata/system_metadata.py

```diff
@@ -195,14 +204,22 @@ class SystemInfo:
 
 
 def get_static_system_info() -> SystemInfo:
-    hostname, distribution, libc_tuple, mac_address, local_ip = _initialize_system_info()
-    clock = getattr(time, "CLOCK_BOOTTIME", time.CLOCK_MONOTONIC)
-    try:
-        spawn_uptime_ms = time.clock_gettime(clock)
-    except OSError as error:
-        if error.errno != errno.EINVAL:
-            raise
-        spawn_uptime_ms = time.clock_gettime(time.CLOCK_MONOTONIC)
+    if is_windows():
+        hostname = platform.node()
+        distribution = platform.system(), platform.release(), platform.version()
+        libc_tuple = platform.libc_ver()
+        mac_address = ""  # Use netifaces pip package
```

Comment:
Let's add this `netifaces`? For mac address & IP

Reply:
Done